resources/js/components/AddressCard.vue

```diff
@@ -4,24 +4,29 @@
             address: Object,
             shipping: Boolean,
             billing: Boolean,
+            customTitle: {
```

Comment:
Is customTitle only used to be passed through this component?
Wouldn't it be better to handle this in the slot itself?

If it's a php variable we can immediately render it, and if it's a js or parent vue component variable we can work with that as well